### ISSUE TITLE
Update the dev policy diagram in the DOD with technical steps

### ISSUE THREAD

**Issue by theblockstalk** [Tonomy-Foundation/Tonomy-ID-Management#122]
Update the DOR and DOD to include the following

1. **[AGREED]** JT propose an implementation plan of what functions and classes changes before starting implementing a new task. this is expected if complicated or you are stuck. the implementation plan exists to have a conversation, but not to fix the outcome at the end.
1. **[AGREED]** JT add to dev process, must rebase and check tests before merge

Acceptance criteria

- [ ] diagram is also updated https://drive.google.com/file/d/1-mACdt8ucz5ONKpqiECjSz3GAP0ASq8o/view?usp=share_link
- [ ] DOD updated/added to CONTRIBUTORS.md updated https://github.com/Tonomy-Foundation/Tonomy-ID-Integration/blob/development/CONTRIBUTING.md#git-and-development-policy

Retrospective notes: https://www.notion.so/tonomy-foundation/320602f5422949a2b814465a5e9354c2?v=6d5b01f1cf4f443cbd519a61c6a20fe6&p=f728b6d1852345bfb4ad122c100398ff&pm=s

**Comment by theblockstalk** [comment]
See https://github.com/Tonomy-Foundation/Tonomy-ID-Integration/blob/feature/definion-of-done-ready/CONTRIBUTING.md#development-process
